includes/admin/settings/class-settings-advanced.php

```diff
@@ -153,6 +153,29 @@ public function get_settings() {
 								'disabled' => __( 'Disabled', 'give' ),
 							],
 						],
+						[
+							'name'    => __( 'Setup Page', 'give' ),
+							/* translators: %s: about page URL */
+							'desc'    => sprintf(
+								wp_kses(
+									__( 'Disable this option if you would like to disable the <a href="%s" target="_blank">GiveWP Setup page</a> that displays when GiveWP is first isntalled.', 'give' ),
+									[
+										'a' => [
+											'href'   => [],
+											'target' => [],
+										],
+									]
+								),
+								esc_url( admin_url( 'edit.php?post_type=give_forms&page=give-setup' ) )
+							),
+							'id'      => 'setup_page_enabled',
+							'type'    => 'radio_inline',
+							'default' => \Give\Onboarding\Onboarding::isSetupPageEnabled(),
+							'options' => [
+								1 => __( 'Enabled', 'give' ),
+								0 => __( 'Disabled', 'give' ),
+							],
```

Comment:
On an install that has not stored anything in the `setup_page_enabled` key (ie updated without running install hook), the `isSetupPageEnabled` method will return `false`. This isn't able to map onto the options of 0 or 1, that are available in the options array, so for users it appears as though nothing has been set. Here's a screenshot for the user experience under those circumstances:

<img width="1123" alt="Screen Shot 2020-07-10 at 11 20 27 AM" src="https://user-images.githubusercontent.com/5186078/87185895-60edb080-c29f-11ea-9707-6946ef9736f0.png">

Reply:
The same issue appears to happen when users dismiss the Setup page. The option value is being stored as a boolean, which isn't able to map onto the options array provided for this particular field (guessing that the JS that controls the field isn't comparing falsey/truthy - it just does a ===).

Reply:
Okay. I'm seeing that when dismissing the Setup Page, but not in a fresh install. I'll take a look.

Reply:
Fixed by mapping the boolean to `1`/`0` to match the option keys.

Reply:
Cool, yeah I think it actually works as expected on a fresh install (because it stores as true on a fresh install), but I ran into the problem when updating a site manually (really sorta developer edge case the more I think about it). Anywayyy, I think the latter will be resolved once if you get it working on page dismissal.